src/libaktualizr/package_manager/packagemanagerconfig.h

```diff
@@ -16,6 +16,10 @@ struct PackageConfig {
 #else
   PackageManager type{PackageManager::kNone};
 #endif
+  // optional, if not specified nothing is updated, just image data are verified and delivered
+  boost::filesystem::path filepath;
+  // optional, if not specified any target can be applied
+  std::string target_name;
```

Comment:
This is just an initial prototype of the file update support on secondary. We have separate/dedicated ticket for this effort, so we can discuss the file update API/configuration then.

Reply:
> We have separate/dedicated ticket for this effort, so we can discuss the file update API/configuration then.

Sure, but then I don't want to add these to the config if we're just going to decide to remove them.

`filepath` seems like it's redundant with `images_path` as used in the SqlStorage. Or am I misunderstanding what it's for?

On the Primary, we've avoided needing `target_name` by just using the device ID as the target name, right? Can't we do something similar for Secondaries?

Reply:
Yes, I can default the target_name to the device-ID, also it won't work if a customer would like to run two aktualizr-secondaries on the same physical ECU.

filepath it's a path to the file a customer would like to update, like firmaware_path in case of Virtual Secondary

Reply:
> it won't work if a customer would like to run two aktualizr-secondaries on the same physical ECU.

Not a very likely use case, and it seems like we ought to be able to work around that without too much effort.

> filepath it's a path to the file a customer would like to update, like firmware_path in case of Virtual Secondary

That makes sense, but I don't think it's a requirement at present to be able to configure this location for secondaries. I think it's fine to just dump it in `/var/sota/images` like we do on the Primary.

If you can't tell, I'm pretty hesitant to add more configuration options without being sure we know what we're doing with them. :)

Reply:
ok, I got your point, will update this PR to make it working without additional params. Also, we might discuss it with the product management to agree on how a file/binray update's configuration/API should be exposed to customers.